src/App.css

```diff
@@ -62,8 +62,8 @@ body {
 
 .button-single {
   background: #9897c2;
-  height: 38px;
-  width: 55px;
+  /*height: 38px;
```

Comment:
Unfortunately this doesn't do anything because of the flex. i've commented it out but it can be removed if you want.